### DISCUSSION TITLE
Would Raspberry Pi 4 sufficient to run a seaweedfs node?

### DISCUSSION THREAD

**Discussion by tigerinus** [seaweedfs/seaweedfs#3732] [Q&A]
Have 4 unused RPi4 at home. As a self-hoster, I wonder what the performance like or about any comments would be for running `seaweedfs` using these RPi4s. Is self-hosted home lab a use case seaweedfs would fit (versus ceph or another alternatievs)? 

Thanks.

**Comment by flkdnt** [comment]
I personally have it running on a 5-node raspberry pi 4 kubernetes cluster. 

I'm still optimizing the configuration as there is a learning curve, but I think I am close to stabilizing it

**Reply by chrislusf** [reply]
How is the performance so far?

**Reply by flkdnt** [reply]
Day-to day usage with the csi-driver has been pretty solid. 

All my k8s nodes have 2GB ram, so I've had to make changes in order to not kill the filer(The Optimization and Production wiki pages have been helpful). 

I'm testing a massive upload with a couple hundred GB of files to see if i get anymore upload errors.

**Comment by kmlebedev** [comment]
And I'm still waiting for the shipment of 4 pre-order [rockpi 5](https://shop.allnetchina.cn/collections/frontpage/products/rock5-model-b?variant=39514839515238) to take benchmark, since you can connect an nvme disk to it :)

**Reply by tigerinus** [reply]
I'd recommend a x86 based SBC, such as zimaboard, which has a real PCI-e bus for better IO performance.
